docs/introduction.md

```diff
@@ -82,12 +82,13 @@ The measurement of one sample is called an MS run and the set of spectra called
 The left image displays spectrum with peaks (m/z and intensity values) and the right image shows spectra stacked in
 retention time yielding a peak map.
 
-In proteomics and metabolomics, the {term}`MS(1)` intensity is often used for the quantification of an analyte. Identification
-based on the {term}`MS(1)` mass-to-charge and the isotope pattern is highly ambiguous. To improve identification, tandem mass
-spectrometry {term}`(MS/MS)` can be applied to assess the analyte substructure. To this end, the precursor ion is isolated and
-kinetically fragmented using an inert gas (e.g., Argon). Fragments produced by {term}`collision-induced dissociation (CID)`
-are stored in an {term}`MS^2` (or {term}`MS/MS`) spectrum and provide information that helps to resolve the ambiguities in identification.
-Alternatively, {term}`MS/MS` spectra can be used for quantification.
+In proteomics and metabolomics, the {term}`MS(1)` intensity is often used for the quantification of an analyte. 
+Identification based on the {term}`MS(1)` mass-to-charge and the isotope pattern is highly ambiguous. To improve 
+identification, tandem mass spectrometry {term}`(MS/MS)` can be applied to assess the analyte substructure. To this end,
+the precursor ion is isolated and kinetically fragmented using an inert gas (e.g., Argon). Fragments produced by {term}
+`collision-induced dissociation (CID)` are stored in an {term}`MS^2` (or {term}`MS/MS`) spectrum and provide information
```

Comment:
why does the user need to know about CID? are there OpenMS algorithms that are based on CID? I think a simplified explanation of what mass spectrometry is, is required.

Reply:
i was reading and felt the need. please feel free, to ticket or create a pull request.